test/persistence-hooks.suite.js

```diff
@@ -1297,6 +1297,26 @@ module.exports = function(dataSource, should) {
             done();
           });
       });
+
+      it('accepts hookState from options', function(done) {
+        TestModel.observe('after save', pushContextAndNext());
+
+        TestModel.updateAll(
+          { id: existingInstance.id },
+          { name: 'updated name' },
+          { hookState: { foo: 'bar' } },
+          function(err) {
+            if (err) return done(err);
+            var expected = { foo: 'bar', test: true };
+            observedContexts.should.eql(aTestModelCtx({
+              where: { id: existingInstance.id },
+              data: { name: 'updated name' },
+              hookState: { foo: 'bar', test: true }
+            }));
```

Comment:
Don't assert on things that are already verified by other tests, keep the test as focused as possible and assert only on the `hookState`.

```js
observedContexts.hookState.should.eql({ foo: 'bar', test: true });
```